Makefile

```diff
@@ -101,16 +101,23 @@ ifneq ($(detected_OS),Windows)
 endif
 
 ifeq ($(detected_OS),Darwin)
-bottles/openssl:
-	./scripts/fetch-brew-bottle.sh openssl
-
-bottles/pcre: bottles/openssl
-	./scripts/fetch-brew-bottle.sh pcre
-
-bottles/hunspell: bottles/pcre
-	./scripts/fetch-brew-bottle.sh hunspell
-
-bottles: bottles/hunspell
+BOTTLES_DIR := $(shell pwd)/bottles
+BOTTLES := $(addprefix $(BOTTLES_DIR)/,hunspell openssl pcre)
+
+$(BOTTLES): | $(BOTTLES_DIR)
+	echo -e "\e[92mFetching:\e[39m $(notdir $@) bottle"
+	./scripts/fetch-brew-bottle.sh $(notdir $@)
+
+$(BOTTLES_DIR):
+	echo -e "\e[92mUpdating:\e[39m macOS Homebrew"
+	if [[ $$(stat -f %u /usr/local/var/homebrew) -ne "$${UID}" ]]; then \
+		echo "Missing permissions to update Homebrew formulae!" >&2; \
+		exit 1; \
```

Comment:
The script never included `exit 1`. Jenkins user does not have permissions to do this. And should not.
```sh
jenkins@macos-01.ms-eu-dublin.ci.misc:~ % stat -f %u /usr/local/var/homebrew
501
jenkins@macos-01.ms-eu-dublin.ci.misc:~ % echo $UID                         
502
```

Reply:
I've updated the `$(BOTTLES_DIR)` target so it has the same `if..else` logic that was in `scripts/fetch-brew-bottle.sh`.

Thinking about the bigger picture re: the original script... what's the motivation for checking the permissions when it's likely to succeed on a developer's machine but not in CI? In other words, on a developer's machine the latest bottle would get downloaded but in CI the bottle version would be relative to whenever `brew update` was last run in that environment. Was that the desired behavior?

Reply:
We already had this conversation: https://github.com/status-im/status-desktop/pull/2258#issuecomment-821094915

Because I'm not giving `jenkins` user permissions to install stuff system-wide.

Reply:
Ah, just thought of something: `brew update` only downloads a catalog, it's `brew upgrade` that installs stuff system-wide.

So maybe downloading a tarball from a newer vs. older catalog is not so much a concern?

Reply:
You can't use `brew update` either:
```
jenkins@macos-01.ms-eu-dublin.ci.misc:~ % brew update
Error: Can't create update lock in /usr/local/var/homebrew/locks!
Fix permissions by running:
  sudo chown -R $(whoami) /usr/local/var/homebrew
jenkins@macos-01.ms-eu-dublin.ci.misc:~ % 
```
Unless I guess you could specify a different location for the database, but that would probably get ugly fast.